test/darwin.js

```diff
@@ -169,6 +169,33 @@ async function appHelpersBundleTest (t, opts, helperBundleId, appBundleId) {
   return assertCFBundleIdentifierValue(t, helperNPObj, helperBundleIdentifier + '.NP', 'CFBundleName should reflect opts.helperBundleId, opts.appBundleId or fallback to default in helper NP app')
 }
 
+async function appHelpersBundleElectron6Test (t, opts, helperBundleId, appBundleId) {
+  if (helperBundleId) {
+    opts.helperBundleId = helperBundleId
+  }
+  if (appBundleId) {
+    opts.appBundleId = appBundleId
+  }
+  opts.electronVersion = '6.0.0-beta.15'
```

Comment:
This Electron version needs to be added to the testcase setup. Although it needs to only download the prebuild `darwin` binary... :cry:

Reply:
I should be clearer here... We pre-download the Electron binaries used so that we don't get timeouts during the actual testsuite run.

Reply:
Added to the initial download list 👍